scripts/deploy/28_samb_treasury_vault.ts

```diff
@@ -30,18 +30,16 @@ const main = async ({
     return;
   }
 
-  const logicDeployment = await deployments.get("RibbonTreasuryVaultLogic");
-  const lifecycle = await deploy("VaultLifecycleTreasuryBare", {
-    contract: "VaultLifecycleTreasuryBare",
-    from: deployer,
-  });
-  console.log(`VaultLifeCycleTreasuryBare @ ${lifecycle.address}`);
+  const logicDeployment = await deployments.get("RibbonTreasuryVaultLogicSAMB");
+  console.log(`LogicDeployment @ ${logicDeployment.address}`);
+  const lifecycleAddress = logicDeployment.libraries.VaultLifecycleTreasuryBare;
```

Comment:
Instead of getting the lifecycle address from the deployment, we just get it directly from the deployed logic contract instead